fidesctl/src/fidesctl/cli/util_comands.py

```diff
@@ -83,17 +82,16 @@ def init(ctx: click.Context, fides_directory_location: str) -> None:
 
 @click.command()
 @click.pass_context
-def ping(ctx: click.Context) -> None:
+def status(ctx: click.Context) -> None:
     """
     Sends a request to the Fidesctl API healthcheck endpoint and prints the response.
     """
     config = ctx.obj["CONFIG"]
-    healthcheck_url = config.cli.server_url + "/health"
-    echo_green(f"Pinging {healthcheck_url}...")
-    try:
-        handle_cli_response(with_analytics(ctx, _api.ping, url=healthcheck_url))
-    except requests.exceptions.ConnectionError:
-        echo_red("Connection failed, webserver is unreachable.")
+    cli_version = fidesctl.__version__
+    server_url = config.cli.server_url
+    echo_green("Getting server status...")
+    check_server(cli_version, server_url)
```

Comment:
We also lost our analytics collection:
```suggestion
    with_analytics(
        ctx,
        check_server,
        cli_version=cli_version,
        server_url=server_url
    )
```